Spinner/Spinner.module.less

```diff
@@ -219,6 +219,16 @@
 					--node-width: @agate-spinner-small-loading-width;
 				}
 			}
+
+			.enact-locale-rtl({
+				.node {
+					right: calc(
+							var(--node-even-spacing) -
+							var(--node-relative-offset) -
+							(var(--node-proportion-offset) * @agate-spinner-loading-width)
+					);
+				}
+			});
```

Comment:
How about moving this code upper to define left for ltr and right for rtl together?
In addition, it might need `left: initial;` to ignore the css style for ltr.

```
				left: calc(
					var(--node-even-spacing) -
					var(--node-relative-offset) -
					(var(--node-proportion-offset) * @agate-spinner-loading-width)
				);

				.enact-locale-rtl({
					right: calc(
						var(--node-even-spacing) -
						var(--node-relative-offset) -
						(var(--node-proportion-offset) * @agate-spinner-loading-width)
					);
					left: initial;
				});
```

If defining a CSS variable for left and right styles, then we could reduce redundant code.
```
				--node-position: calc(
					var(--node-even-spacing) -
					var(--node-relative-offset) -
					(var(--node-proportion-offset) * @agate-spinner-loading-width)
				);

				left: var(--node-position);

				.enact-locale-rtl({
					right: var(--node-position);
					left: initial;
				});
```

Reply:
I see. Thanks for suggestion!